doc/classes/TreeItem.xml

```diff
@@ -499,6 +543,7 @@
 			<argument index="1" name="selectable" type="bool">
 			</argument>
 			<description>
+				Sets whether the column [code]column[/code] is selectable.
```

Comment:
Per writing guide, boolean methods prefer true/false wording over "whether". Try:

    If [code]true[/code] the given column is selectable.